owncloudApp/src/main/java/com/owncloud/android/workers/CameraUploadsWorker.kt

```diff
@@ -202,8 +205,10 @@ class CameraUploadsWorker(
         )
     }
 
-    private fun updateTimestamp(folderBackUpConfiguration: FolderBackUpConfiguration, syncType: SyncType) {
-        val currentTimestamp = System.currentTimeMillis()
+    private fun updateTimestamp(
+        folderBackUpConfiguration: FolderBackUpConfiguration, syncType: SyncType,
+        currentTimestamp: Long
```

Comment:
Here too. And maybe each parameter in a separated line?

Reply:
Sometimes automatic code formatting doesn't work well... 🤕